content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -4,19 +4,22 @@ This article is about Sockets. It will give you a glimpse of how clients and ser
 
 Socket programming is a means of communicating data between two computers across a network. Connections can be made using either a **connection-oriented protocol** or a **connectionless protocol**. TCP/IP, a connection-oriented protocol, will be used in our case.
 
-Before exchanging data, two computers must establish a link in connection-oriented protocols. In contrast, you don't require every packet to arrive at its intended destination for connectionless. It is made feasible via UDP (User Datagram Protocol). To demonstrate sockets further, we shall use the Client\Server architecture. Client and server communicate by writing to and reading from the socket connection. The machines must have each other's network information to make this connection.
+Before exchanging data, two computers must establish a link in connection-oriented protocols. In contrast, you don't require every packet to arrive at its intended destination for connectionless. It is made feasible via UDP (User Datagram Protocol).
 
-Table of contents:
-- [Definition-of-a-Socket](#definition-of-a-socket)
-- [Programming-on-the-Server-side ](#programming-on-the-server-side)
-- [Programming-on-the-Client-side](#programming-on-client-side)
-- [How-to-Run-Programs-in-a-Variety-of-Applications](#how-to-run-programs-in-a-variety-of-applications)
+To demonstrate sockets further, we shall use the Client/Server architecture. Client and server communicate by writing to and reading from the socket connection. The machines must have each other's network information to make this connection.
+
+### Table of contents:
+- [Definition of a Socket](#definition-of-a-socket)
+- [Programming on the Server-side ](#programming-on-the-server-side)
+- [Programming on the Client-side](#programming-on-client-side)
```

Comment:
> \[Programming on the Client-side\](\#programming-on-client-side)

This link doesn't work. Change it to this to this and it should work:

> \[Programming on the Client-side\](\#programming-on-the-client-side)

Please verify that it indeed works when making this change. Refer to this [StackOverflow post](https://stackoverflow.com/questions/51221730/markdown-link-to-header) if you need help with the links.